core/runtime/tests/misc.rs

```diff
@@ -1058,41 +1050,34 @@ async fn tla_in_esm_extensions_panics() {
 #[tokio::test]
 #[should_panic(
   expected = r#"Failed to initialize a JsRuntime: Error: This fails
-    at a (mod:error:2:34)
-    at mod:error:3:13"#
+    at a (mod:error:2:30)
+    at mod:error:3:9"#
 )]
 async fn esm_extensions_throws() {
   #[op2(async)]
   async fn op_wait(#[number] ms: usize) {
     tokio::time::sleep(Duration::from_millis(ms as u64)).await
   }
 
-  let extension = Extension {
-    name: "test_ext",
-    ops: Cow::Borrowed(&[op_wait::DECL]),
-    esm_files: Cow::Borrowed(&[
-      ExtensionFileSource {
-        specifier: "mod:error",
-        code: ExtensionFileSourceCode::IncludedInBinary(
-          r#"
-            function a() { throw new Error("This fails") };
-            a();
-          "#,
-        ),
-      },
-      ExtensionFileSource {
-        specifier: "mod:test",
-        code: ExtensionFileSourceCode::IncludedInBinary("import 'mod:error';"),
-      },
-    ]),
-    esm_entry_point: Some("mod:test"),
-    ..Default::default()
-  };
+  deno_core::extension!(
+    test_ext,
+    ops = [op_wait],
+    esm_entry_point = "mod:test",
+    esm = [
+      "mod:test" = { source = "import 'mod:error';" },
+      "mod:error" = {
+        source = r#"
+        function a() { throw new Error("This fails") };
+        a();
```

Comment:
These will have leading spaces, right?

Reply:
Correct. All of these embedded raw strings have an extra indent.

Reply:
Guess we're gonna use them mostly for testing, so that seems fine.